addons/mail/static/tests/messaging_menu/messaging_menu_tests.js

```diff
@@ -175,6 +180,67 @@ QUnit.test("rendering with PWA installation request", async (assert) => {
     assert.verifySteps(["show prompt"]);
 });
 
+QUnit.test("installation of the PWA request can be dismissed", async (assert) => {
+    const target = getFixture();
+
+    patchWithCleanup(browser, {
+        BeforeInstallPromptEvent: () => {},
+    });
+    patchWithCleanup(browser.localStorage, {
+        getItem(key) {
+            if (key === "pwa.installationState") {
+                assert.step("getItem " + key);
+                // in this test, installation has not yet proceeded
+                return null;
+            }
+            return super.getItem(key);
+        },
+        setItem(key, value) {
+            if (key === "pwa.installationState") {
+                assert.step("installationState value:  " + value);
+            }
+            return super.setItem(key, value);
+        },
+    });
+
+    const { env } = await start();
+    // This event must be triggered to initialize the installPrompt service properly
+    // as if it was run by a browser supporting PWA (never triggered in a test otherwise).
+    window.dispatchEvent(new CustomEvent("beforeinstallprompt"));
+
+    patchWithCleanup(env.services.installPrompt, {
+        show() {
+            assert.step("show prompt should not be triggered");
+        },
+    });
+    assert.verifySteps(["getItem pwa.installationState"]);
+
+    await contains(".o-mail-MessagingMenu-counter");
+    await contains(".o-mail-MessagingMenu-counter", { text: "1" });
+    await click(".o_menu_systray i[aria-label='Messages']");
+    await contains(".o-mail-NotificationItem");
+    assert.ok(
+        target
+            .querySelector(".o-mail-NotificationItem img")
+            .dataset.src.includes("/web/image?field=avatar_128&id=2&model=res.partner")
+    );
+    assert.strictEqual(
+        target.querySelector(".o-mail-NotificationItem-name").textContent,
+        "OdooBot has a suggestion"
+    );
+    assert.strictEqual(
+        target.querySelector(".o-mail-NotificationItem-text").textContent,
+        "Come here often? Install Odoo on your device!"
+    );
+
+    await click(".o-mail-NotificationItem .fa-close");
+    assert.verifySteps(["installationState value:  dismissed"]);
+
+    await nextTick();
+    await click(".o_menu_systray i[aria-label='Messages']");
```

Comment:
`nextTick()` looks useless

Reply:
I know. But without it, the list of notifications is not rerended yet, and the test fails :/